boards/rapidsilicon/virgo_proto/doc/index.rst

```diff
@@ -0,0 +1,240 @@
+.. virgo:
+
+Rapidsi Virgo
+####################
+
+Overview
+********
+
+.. image:: img/virgo.jpg
+     :align: center
+     :alt: Virgo
+
+More information can be found on ' ' websites.
+
+Hardware
+********
+
+Summary:
+
+The virgo platform provides following hardware components:
+
+- 1 core 32-bit 266MHz AndeStar v5 (N22) RISC-V CPU
+- 64KB on-board ILM
```

Comment:
```suggestion
- 32KB on-board ILM
```
As per the SoC_uarch, this is 32 KB, can you please confirm?

Reply:
The DLM is 32KB and ILM, where we load fsbl, is 64KB. That is what we have been using in soc prototype. If in Test chip they have 32KB of ILM then we will be in a very bad shape.

Reply:
I think the SOC ARCH is incorrect, as per the virgo RTL code the ILM is 64K
![image](https://github.com/os-fpga/zephyr_rs/assets/104416282/b52c5f77-d745-472a-9e67-4304ddd35127)